### PR TITLE
Add roadmap styles

### PR DESCRIPTION
**Pull request by kgabryje**

### CODE REVIEW

File: landing-pages/src/js/progressTracking.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const contentRoot = window.document.querySelector(".rst-content");
+const sideNavRoot = window.document.querySelector(".wy-nav-side-toc");
+
+let currentActive = null;
+
+const runProgressTracking = () => {
+  if (!contentRoot || !sideNavRoot) return;
+
+  const sectionsOffsets = [];
+  contentRoot.querySelectorAll(".section").forEach((element) => sectionsOffsets.push({
```
Comment (mik-laj) [inline landing-pages/src/js/progressTracking.js]:
```suggestion
  const sectionsOffsets = contentRoot.querySelectorAll(".section").map((element) => ({
```

File: landing-pages/src/js/progressTracking.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    id: element.id,
+    offset: element.offsetTop
+  }));
+
+  const handleActiveSection = () => {
+    const lastActive = currentActive;
+    const currentActiveIndex = Math.max(
+      sectionsOffsets.findIndex((section) => section.offset - window.scrollY + 123 > 0) - 1,
```
Comment (mik-laj) [inline landing-pages/src/js/progressTracking.js]: What is 123? Can you explain magic constant?

File: landing-pages/src/js/progressTracking.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      const anchor = sideNavRoot.querySelector(`[href='#${lastActive.id}']`);
+      anchor && anchor.closest("li").classList.remove("current");
+    }
+    if (currentActive) {
+      const anchor = sideNavRoot.querySelector(`[href='#${currentActive.id}']`);
+      anchor.closest("li").classList.add("current");
+    }
+  };
+
+  window.addEventListener("scroll", debounce(handleActiveSection, 100));
```
Comment (mik-laj) [inline landing-pages/src/js/progressTracking.js]:
```suggestion
  window.addEventListener("scroll", debounce(handleActiveSection, 100));
  window.addEventListener("resize", debounce(handleActiveSection, 100));
```

File: landing-pages/src/js/progressTracking.js (hunk 2 above)
Comment (kgabryje) [reply landing-pages/src/js/progressTracking.js]: done

**Pull request merged**